src/main/support.js

```diff
@@ -9,30 +9,26 @@ const arrayify = require("../utils/arrayify");
 const currentVersion = require("../../package.json").version;
 const coreOptions = require("./core-options").options;
 
-function getSupportInfo(version, opts) {
-  opts = {
-    plugins: [],
-    showUnreleased: false,
-    showDeprecated: false,
-    showInternal: false,
-    ...opts
-  };
-
-  if (!version) {
-    // pre-release version is smaller than the normal version in semver,
-    // we need to treat it as the normal one so as to test new features.
-    version = currentVersion.split("-", 1)[0];
+function getSupportInfo(
+  _,
+  // This has to be the second parameter. See _withPlugins in src/index.js.
+  {
+    plugins = [],
+    showUnreleased = false,
+    showDeprecated = false,
+    showInternal = false
   }
-
-  const { plugins } = opts;
+) {
+  // pre-release version is smaller than the normal version in semver,
+  // we need to treat it as the normal one so as to test new features.
+  const version = currentVersion.split("-", 1)[0];
```

Comment:
I never know split takes a second argument.

Reply:
It's not super useful anyway.